dynawo/sources/Modeler/DataInterface/PowSyblIIDM/DYNDataInterfaceIIDM.cpp

```diff
@@ -679,6 +680,7 @@ DataInterfaceIIDM::convertThreeWindingsTransformers(powsybl::iidm::ThreeWindings
     }
     network_->addTwoWTransformer(fictTwoWTransf);
     components_[fictTwoWTransf->getID()] = fictTwoWTransf;
+    fict2wtIDto3wtID_.insert({fictTwoWTransf->getID(), ThreeWindingTransformer.getId()});
```

Comment:
I know it's should have been already done, but could you please add a unit test for those 2 methods?